src/vmm/tests/cpu_templates.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::path::PathBuf;
```

Comment:
nit: this file introduces a bit more than "remove unnecessary comments"

Reply:
ah sorry, it seems the first commit "refactor(vmm): Remove unnecessary comments" contains some contents that I used in debug.

Reply:
removed the polluted part from the first commit.